src/frontend/packages/auth-provider/src/authProvider.js

```diff
@@ -227,6 +227,51 @@ const authProvider = ({
     } catch (e) {
       throw new Error('app.notification.new_password_error');
     }
+  },
+  getAccountSettings: async params => {
+    const serverUrl = middlewareUri || (params.domain && `https://${params.domain}/`);
+    if (!serverUrl)
+      throw new Error(
+        'You must specify a middlewareUri in the authProvider config, or specify a domain when calling the getAccountSettings method'
+      );
+      
+    try {
+      const token = localStorage.getItem('token');
+      const { webId } = jwtDecode(token);
+      const { json } = await httpClient(`${serverUrl}auth/account?webId=${webId}`, {
+        method: 'GET',
+      });
```

Comment:
You do not need to pass the webId, it is automatically retrieved by the middleware through the JWT (which is automatically sent by httpClient) and available as `ctx.meta.webId`. It's better for the security as well.

Reply:
(The default method is GET so you don't need to define it)

Reply:
~Thank you for this suggestion !

I've modified the `authProvider` and the `account` service to use the `webId` from `ctx.meta` instead of params.